growsocial/client/views/login/login.js

```diff
@@ -103,7 +117,16 @@ Template.register.events({
         console.log('createUser error message:', err.message);
         template.messages.set('infoMessage', null);
         template.messages.set('errorMessage', err.message);
-        // TODO log the error in the database, unless it's "unable to connect" error
+
+        // log the error in the database, unless it's "unable to connect" error
+        if (err.message == "unable to connect") {
```

Comment:
The TODO was a little misleading, please remove the if condition and log any errors as normal.